src/include/libpbs.h

```diff
@@ -332,7 +344,9 @@ int PBSD_jscript_direct(int, char *, int, char **);
 int PBSD_copyhookfile(int, char *, int, char **);
 int PBSD_delhookfile(int, char *, int, char **);
 int PBSD_mgr_put(int, int, int, int, char *, struct attropl *, char *, int, char **);
+int PBSD_delete_put(int, int, int, int, char **, struct attropl *, char *, int, char **);
```

Comment:
I suggest to rename this to say its for deljoblist like PBSD_deljoblist_put

Reply:
okay, I agree

Reply:
@bremanandjk  this hasn't been addressed yet